### DISCUSSION TITLE
Thanks a lot for putting together these courses :)

### DISCUSSION THREAD

**Discussion by AlexConnat** [OpenMined/courses#19] [Introduce Yourself]
Hello everyone 👋

My name is Alex Connat, until last year I was a student at [EPFL](https://www.epfl.ch/en/) 🇨🇭. I completed my Master's degree in Information Security, and worked as a security engineer 👨🏻‍💻 for a while, but I've always been interested in Privacy both from a social and technological perspective. That's why I did my Master Thesis on the topic of Privacy Engineering (building a decentralized differentially-private machine learning classifier ; more information [on my blog](https://alexconnat.github.io/posts/master-thesis/)), and I try to keep up with the current academic literature in this field.

Most importantly, I'm very friendly (please come talk on [Slack](slack.openmined.org) / [LinkedIn](https://ch.linkedin.com/in/connat)) and I looove to learn new stuff, so I really look forward to attend the class, see you there! 😊

**Comment by iamtrask** [comment]
Hi Alex! Congratulations on completing your masters! EPFL has a great program. Great to meet you and I look forward to seeing you around the community :)

**Comment by em-blue** [comment]
Thanks, Alex! :) Hope you enjoy the course. Let us know if you have any feedback about the topics, we're always here to have a chat about it
